config/telegraph.php

```diff
@@ -63,4 +63,11 @@
         'bot' => DefStudio\Telegraph\Models\TelegraphBot::class,
         'chat' => DefStudio\Telegraph\Models\TelegraphChat::class,
     ],
+
+    /**
+     * The language in which the bot will write in case of errors / other system events
+     */
+    'system_alert_lang' => 'en',
```

Comment:
I think this is not necessary anymore, the language should be deducted from the laravel app language or deducted by user code and set up with `App::setLocale(...)`